Packs/Anomali_ThreatStream/Integrations/AnomaliThreatStreamv3/AnomaliThreatStreamv3.py

```diff
@@ -2538,6 +2538,40 @@ def search_intelligence(client: Client, **kwargs):
     )
 
 
+def add_indicator_tag_command(client: Client, **kwargs) -> CommandResults:
+    indicator_ids: list[str] = argToList(kwargs["indicator_ids"])
+    tags: list[str] = argToList(kwargs["tags"])
+
+    data_request = {
+        "ids": indicator_ids,
+        "tags": [{"name": tag, "tlp": "red"} for tag in tags],
+    }
+
+    client.http_request("POST", "v2/intelligence/bulk_tagging/", data=json.dumps(data_request))
```

Comment:
Let's put this part in a client method, that only accepts `indicator_ids`, `tags`